local-cli/bundle/bundleCommandLineArgs.js

```diff
@@ -18,8 +18,7 @@ module.exports = [
     default: 'ios',
   }, {
     command: '--transformer [string]',
-    description: 'Specify a custom transformer to be used (absolute path)',
-    default: require.resolve('../../packager/transformer'),
```

Comment:
Looks reasonable and simple that I think I can merge this. Just one question - why remove the default here? Could it break any code relying on the default?